internal/controller/controllers/agent_controller.go

```diff
@@ -349,8 +349,8 @@ func (r *AgentReconciler) populateEventsURL(log logrus.FieldLogger, agent *aiv1b
 	return nil
 }
 
-func (r *AgentReconciler) eventsURL(log logrus.FieldLogger, clusterId, infraEnvId, agentId string) (string, error) {
-	eventsURL := fmt.Sprintf("%s%s/v1/clusters/%s/events?host_id=%s", r.ServiceBaseURL, restclient.DefaultBasePath, clusterId, agentId)
+func (r *AgentReconciler) eventsURL(log logrus.FieldLogger, infraEnvId, agentId string) (string, error) {
```

Comment:
```suggestion
func (r *AgentReconciler) generateEventsURL(log logrus.FieldLogger, infraEnvId, agentId string) (string, error) {
```

Reply:
Done in https://github.com/openshift/assisted-service/pull/3111